src/profile/components/multiItemAddressRow/MultiItemAddressRow.tsx

```diff
@@ -0,0 +1,189 @@
+import { TextInput } from 'hds-react';
+import React from 'react';
+import { Field, Formik, FormikProps, Form } from 'formik';
+import { useTranslation } from 'react-i18next';
+import countries from 'i18n-iso-countries';
+import classNames from 'classnames';
+
+import commonFormStyles from '../../../common/cssHelpers/form.module.css';
+import { AddressValue } from '../../helpers/editData';
+import { createFormFieldHelpers } from '../../helpers/formik';
+import { addressSchema } from '../../../common/schemas/schemas';
+import FormButtons from '../formButtons/FormButtons';
+import EditButtons from '../editButtons/EditButtons';
+import FormikDropdown, {
+  HdsOptionType,
+} from '../../../common/formikDropdown/FormikDropdown';
+import getLanguageCode from '../../../common/helpers/getLanguageCode';
+import LabeledValue from '../../../common/labeledValue/LabeledValue';
+import getCountry from '../../helpers/getCountry';
+import { getFormFields } from '../../helpers/formProperties';
+import SaveIndicator from '../saveIndicator/SaveIndicator';
+import { useCommonEditHandling } from '../../hooks/useCommonEditHandling';
+import { RowItemProps } from '../multiItemEditor/MultiItemEditor';
+
+type FormikValues = AddressValue;
+
+function MultiItemAddressRow(props: RowItemProps): React.ReactElement {
+  const { data, testId, dataType } = props;
+  const value = data.value as AddressValue;
+  const { address, city, postalCode, countryCode } = value;
+  const { t, i18n } = useTranslation();
+  const lang = i18n.languages[0];
+  const applicationLanguage = getLanguageCode(i18n.languages[0]);
+  const countryList = countries.getNames(applicationLanguage);
```

Comment:
Would it be good idea to sort the countries based on names? Most of them seemed to be sorted already, but I found in the end of the list there's countries that are not sorted with the rest of the countries?
![image](https://user-images.githubusercontent.com/2777633/116708223-2a248a80-a9d8-11eb-9d51-b1c96aa92f8c.png)

Reply:
What language is your browser or profile? I've different ordering.

Reply:
If UI language is English, the sorting is messed up. In Finnish version, only "Kosovo" is in wrong place. Ill' sort them manually.

Reply:
Added sorting